delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutes.scala

```diff
@@ -158,6 +168,9 @@ final class ResolversRoutes(identities: Identities, acls: Acls, projects: Projec
                           }
                         )
                       }
+                    },
+                    iri { resourceIri =>
```

Comment:
why is here an iri? It should be an `idSegment` since it is the same as the other API operations related with ids

Reply:
It can't be an IdSegment because you don't have the mappings and the base to expand it properly as an Iri.

Reply:
Why not? you use the mappings and the base of the current project

Reply:
The resource may be in another project

Reply:
yes, but the API is centered on the original project, from the API perspective that's what matters